packages/contracts/src/dollar/libraries/LibUbiquityPool.sol

```diff
@@ -302,8 +302,12 @@ library LibUbiquityPool {
         view
         returns (uint256 dollarPriceUsd)
     {
+        // load storage shared across all libraries
+        AppStorage storage store = LibAppStorage.appStorage();
         // get Dollar price from Curve Metapool (18 decimals)
-        uint256 dollarPriceUsdD18 = LibTWAPOracle.getTwapPrice();
+        uint256 dollarPriceUsdD18 = ICurveStableSwapMetaNG(
```

Comment:
Pairing Dollar with the tripool will still cause [this issue](https://github.com/sherlock-audit/2023-12-ubiquity-judging/issues/15), even if the pool used is a StableSwapMetaNG and not the old curve pools. 
Recommendation:
The best solution to achieve an actual peg to $1 would be to use Chainlink price feeds instead. or to pair Dollar with a different asset that is actually pegged to $1 rather than 3CRV, given that currently the pool price will tend to follow 3CRV's price, which currently trades at $1.03.